### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by v-abraham**
Update the game over message so people know how to play again!

### CODE REVIEW

File: index.html
Comment (v-abraham) [inline index.html:1]: Index Comment

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
```
Comment (v-abraham) [inline index.html:49]:
```suggestion
Test Suggestion
```

**Review by tara-abraham — APPROVED** [review]
Seems not ready,add

**Review by tara-abraham — CHANGES_REQUESTED** [review]
needs more testing

**Pull request merged**